atomics/T1078.004/T1078.004.yaml

```diff
@@ -49,6 +48,12 @@ atomic_tests:
       if [ -x "$(command -v gcloud)" ]; then exit 0; else exit 1; fi;
     get_prereq_command: |
       echo "Please Install Google Cloud SDK before running this atomic test : https://cloud.google.com/sdk/docs/install"
+  - description: | 
+      Check if user is logged in 
+    prereq_command: |
+      gcloud config get-value account
+    get_prereq_command: |
+      gcloud auth login --no-launch-browser
```

Comment:
Moved login as a `dependency` so that authentication is not triggered for every atomic.

Reply:
T1098 Execution
<img width="1472" alt="T1098-1" src="https://github.com/redcanaryco/atomic-red-team/assets/27735081/137cb781-d6e5-4be7-9711-cdfeeb218b91">
<img width="1472" alt="T1098-2" src="https://github.com/redcanaryco/atomic-red-team/assets/27735081/90af2df9-9800-4e0e-ae10-a898d2366084">

Reply:
T1485 Execution
<img width="1472" alt="T1485-1" src="https://github.com/redcanaryco/atomic-red-team/assets/27735081/00abd00d-f34e-4e23-b380-d1438b8f853f">
<img width="1472" alt="T1485-2" src="https://github.com/redcanaryco/atomic-red-team/assets/27735081/9965839e-e7cc-4a81-a524-ce8fcb63e94b">